index.js

```diff
@@ -55,6 +55,56 @@ server.get("/login", (request, response) => {
   response.sendFile(__dirname+"/views/login.html");
 });
 
+server.get("/cadastroUser", (request, response) => {
+  var emailMsn = "Insira um email:";
+  var password1Msn = "Insira sua senha:";
+  var password2Msn = "Confirme a senha:";
```

Comment:
Quando o conteúdo da variável nunca muda, prefira usar `const `em vez de `var` e mesmo quando ela mudar, prefira `let` em vez de `var`
```suggestion
  var emailMsn = "Insira um email:";
  var password1Msn = "Insira sua senha:";
  var password2Msn = "Confirme a senha:";
```

Da uma olhada nesse [artigo ](https://www.alura.com.br/artigos/entenda-diferenca-entre-var-let-e-const-no-javascript)explicando as diferenças